ci-operator/config/openshift/secrets-store-csi-driver-operator/openshift-secrets-store-csi-driver-operator-main.yaml

```diff
@@ -98,30 +98,57 @@ tests:
   commands: make verify
   container:
     from: src
-- as: operator-e2e
+- as: operator-e2e-fips
   steps:
     cluster_profile: azure4
     dependencies:
       OO_INDEX: ci-index-secrets-store-csi-driver-operator-bundle
       SECRETS_STORE_E2E_PROVIDER_IMAGE: secrets-store-csi-driver-e2eprovider
     env:
+      FIPS_ENABLED: "true"
       OO_CHANNEL: stable
       OO_INSTALL_NAMESPACE: openshift-cluster-csi-drivers
       OO_PACKAGE: secrets-store-csi-driver-operator
       OO_TARGET_NAMESPACES: '!all'
+    test:
+    - ref: fips-check-fips-or-die
+    - ref: fips-check-node-scan
+    - as: test-e2e
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 50m
+          memory: 100Mi
```

Comment:
We need to copy `test-e2e` target from the workflow to the ci-operator configuration to avoid override.

In cases where both the workflow and a ci-operator configuration
specify the same field (e.g test), the ci-operator configuration's field has priority.

- https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration
- https://docs.ci.openshift.org/docs/architecture/step-registry/#hierarchical-propagation